lib/onebox/engine/twitter_status_onebox.rb

```diff
@@ -27,10 +23,46 @@ def to_html
 
       private
 
+      def get_twitter_data
+        response =
+          begin
+            Onebox::Helpers.fetch_response(url, headers: http_params)
+          rescue StandardError
+            nil
```

Comment:
Should we exit early here? Also, can we add a 404 response case as well?

Reply:
Yes, we could, but what should we do differently in that case? Log a warning? That'll get noisy, I think.

Reply:
Yeah, just return early I guess? We can also add logging behind a debug flag... but sounds like overkill indeed.